ThermoPower/package.mo

```diff
@@ -2105,6 +2105,6 @@ This is a major new release, that has been in the making for 5 years. The new re
 <p>The ThermoPower package is licensed by Politecnico di Milano under the <b><a href=\"http://www.modelica.org/licenses/ModelicaLicense2\">Modelica License 2</a></b>. </p>
 <p><h4>Copyright &copy; 2002-2014, Politecnico di Milano.</h4></p>
 </html>"),
-  uses(Modelica(version="3.2.2")),
+  uses(Modelica(version="3.2.3")),
```

Comment:
This change is probably a good idea, since MSL 3.2.3 is used by all up-to-date Modelica tools as of today. I  would suggest to make a separate pull request for this one, for documentation purposes